src/plopp/backends/pythreejs/mesh3d.py

```diff
@@ -0,0 +1,179 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2024 Scipp contributors (https://github.com/scipp)
+
+import uuid
+from typing import Literal
+
+import numpy as np
+import scipp as sc
+from matplotlib.colors import to_rgb
+
+from ...core.limits import find_limits
+from ...graphics.bbox import BoundingBox
+from .canvas import Canvas
+
+
+class Mesh3d:
+    """
+    Artist to represent a three-dimensional mesh.
+
+    Parameters
+    ----------
+    canvas:
+        The canvas to draw the mesh on.
+    data:
+        The initial data to create the mesh from. Must be a DataGroup that contains at
+        least the following fields:
+        - vertices: a DataArray with the vertices of the mesh.
+        - faces: a DataArray with the faces of the mesh.
+    color:
+        The color of the mesh. If None, the mesh will be colored according to the
+        artist number.
+    opacity:
+        The opacity of the mesh.
+    edgecolor:
+        The color of the edges of the mesh. If None, the edges will not be shown.
+    artist_number:
+        The number of the artist. This is used to determine the color of the mesh if
+        `color` is None.
+    """
+
+    def __init__(
+        self,
+        *,
+        canvas: Canvas,
+        data: sc.DataArray,
+        color: str | None = None,
+        opacity: float = 1,
+        edgecolor: str | None = None,
+        artist_number: int = 0,
+    ):
+        import pythreejs as p3
```

Comment:
What if pythreejs is not installed or python is not running in Jupyter? Does the backend manager raise an appropriate exception or does this need to happen here?

Reply:
No idea. It's the same in `scatter3d`.
I don't think it's an issue not running in Jupyter, as the unit tests pass fine.

If it's not installed, you will just get `ImportError` on `pythreejs`?

Reply:
> If it's not installed, you will just get ImportError on pythreejs?

I would assume so. Maybe this is something that the backend manager can catch so we don't have to handle it in everywhere. But that is unrelated to this PR.